client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -106,6 +107,9 @@ const availableFlows: Record< string, () => Promise< { default: Flow } > > = {
 	[ DOMAIN_TRANSFER ]: () =>
 		import( /* webpackChunkName: "domain-transfer" */ './domain-transfer' ),
 
+	[ GOOGLE_DOMAIN_TRANSFER ]: () =>
+		import( /* webpackChunkName: "domain-transfer" */ './google-domain-transfer' ),
```

Comment:
I'm unsure of the best approach to name the webpack chunk. Group with domain-transfer made sense to me since there's a lot of overlap.

I'm open to feedback here.